### PR TITLE
New-AzDiskConfig cmdlet with new parameters Tier and Sectorsize 

### PR DESCRIPTION
**Pull request by Sandido**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by Sandido** [comment]
adding help doc and changelog changes shortly.

File: src/Compute/Compute.Test/ScenarioTests/DiskRPTests.ps1 [hunk 1]
```diff
@@ -1168,3 +1168,47 @@ function Test-SnapshotConfigDiskAccessNetworkPolicy
...
 }
+
+<#
+.SYNOPSIS
+Testing the new parameters 
+Tier
+LogicalSectorSize 
+in the New-AzDiskConfig cmdlet.  
+#>
+function Test-DiskConfigTierSectorSizeReadOnly
```
Comment (Sandido) [inline src/Compute/Compute.Test/ScenarioTests/DiskRPTests.ps1:1179]: This test will also have the New-AzDiskUpdateConfig changes with the ReadOnly parameters, hence the ReadOnly part of the name.

File: src/Compute/Compute.Test/ScenarioTests/DiskRPTests.ps1 [hunk 2]
```diff
@@ -1168,3 +1168,47 @@ function Test-SnapshotConfigDiskAccessNetworkPolicy
...
+
+            Assert-AreEqual $diskTier.Tier $tier3; 
+            
+            $diskSector = New-AzDiskConfig -Location $loc -DiskSizeGB 5 `
+                -SkuName UltraSSD_LRS -OsType Windows -CreateOption Empty -LogicalSectorSize $sectorSize `
+                | New-AzDisk -ResourceGroupName $rgname -DiskName $diskNameSector;
+
+            Assert-AreEqual $diskSector.CreationData.LogicalSectorSize $sectorSize; 
+
+		}
```
Comment (Sandido) [inline src/Compute/Compute.Test/ScenarioTests/DiskRPTests.ps1:1207]: Not sure why the } are indented incorrectly here. They were fine in VS.

**Comment by grizzlytheodore** [comment]
Looks good to me! merge as you wish

**Review by grizzlytheodore — APPROVED** [review]

**Pull request merged**